p2p/switch.go

```
@@ -298,9 +298,9 @@ func (sw *Switch) startInitPeer(peer *peer) {
 
 // DialSeeds dials a list of seeds asynchronously in random order.
 func (sw *Switch) DialSeeds(addrBook *AddrBook, seeds []string) error {
```

Cool, so maybe this method doesn't even need to return an error any more :P